### DISCUSSION TITLE
cant catch "grenade_bounce , grenade_thrown , item_purchase..."

### DISCUSSION THREAD

**Discussion by Dmytro4** [saul/demofile#244] [Questions] [answered]
I tried to catch those events but i didnt get anything with
```
   demoFile.gameEvents.on(event_name, e => {
	   ...
   });
```

**Comment by saul** [comment, marked as answer]
Hi @Dmytro4 -

Some events are not networked, they are server-side only. I believe the events you're referring to are not networked and so will not be in the demo.
